src/libs/kdb/symbols.map

```diff
@@ -0,0 +1,53 @@
+libelektra_0.8 {
+	elektraModulesClose;
+	elektraModulesInit;
+	elektraModulesLoad;
+
+	elektraPluginClose;
+	elektraPluginGetFunction;
+	elektraPluginOpen;
+
+	kdbClose;
+	kdbGet;
+	kdbOpen;
+	kdbSet;
+};
+
+libelektra_1.0 {
+	elektraGOptsContract;
+	elektraGOptsContractFromStrings;
+
+	# FIXME: should be separate lib
+	# kdbchangetracking.h
+	elektraChangeTrackingGetContextFromKdb;
+	elektraChangeTrackingGetContextFromPlugin;
+	elektraChangeTrackingCalculateDiff;
+
+	# FIXME: separate lib? part of merge?
```

Comment:
I don't see why diff should be a part of merge TBH. A separate lib maybe, but on the other hand it's an integral part of KDB so IDK.

Reply:
The change tracking part is integral to KDB, but the pure diff part could be used standalone (just with core), AFAIK. I thought of merge, since diff & merge are somewhat related concepts.